config/application.yml.example

```diff
@@ -7,56 +7,68 @@
 # makes 'Your_Gmail_Username' available as ENV["GMAIL_USERNAME"]
 # Add application configuration variables here, as shown below.
 
-development:
+test:
```

Comment:
That file didn't really change. I only aligned the different environments with the same envs + removed some that were in use.